specification/kubernetesruntime/KubernetesRuntime.Management/service.tsp

```diff
@@ -0,0 +1,37 @@
+import "./common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+
+namespace Microsoft.KubernetesRuntime;
+
+@doc("Properties for the service resource")
+model ServiceProperties {
+  @doc("The object id of the service principal of the RP provisioned in the tenant")
+  @visibility("read")
+  rpObjectId?: string;
+
+  @doc("Resource provision state")
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+@doc("A Service resource")
+model ServiceResource is ExtensionResource<ServiceProperties> {
```

Comment:
>ExtensionResource [](http://example.com/codeflow?start=25&length=17)

What all types can it extend? You should try to document the expected usage scenarios